jobs/models.py

```diff
@@ -25,51 +23,112 @@ def __unicode__(self):
-    description = models.TextField(max_length=5000)
+    description = models.TextField()
     reviewer = models.ForeignKey(
-        User, 
+        User,
         related_name="jobs",
         blank=True,
         null=True,
         on_delete=models.SET_NULL
     )
-    review_status = models.BooleanField(default=False, help_text="Check if reviewed")
-    reviewers_comment = models.TextField(max_length=5000, blank=True, null=True)
+    review_status = models.BooleanField(default=False,
+                                        help_text="Check if reviewed")
+    reviewers_comment = models.TextField(blank=True, null=True)
     ready_to_publish = models.BooleanField(default=False)
     published_date = models.DateTimeField(blank=True, null=True)
     created = models.DateTimeField(auto_now_add=True)
     expiration_date = models.DateField(
         blank=True,
         null=True,
-        help_text="Automatically is set 60 days from posting. You can override this."
+        help_text="Automatically is set 60 days from posting. You can"
+                  " override this."
     )
 
-
     class Meta:
         unique_together = (("company", "title"),)
         ordering = ['-published_date']
 
     def publish(self):
-        if self.ready_to_publish:
-            self.published_date = timezone.now()
-            self.save()
-
-    def set_expiration_date(self):
-        if self.published_date:
-            if self.ready_to_publish and not self.expiration_date:
-                self.expiration_date = self.published_date + timedelta(60)
-                self.save()
-            if self.ready_to_publish and self.expiration_date:
-                self.expiration_date = self.published_date + timedelta(60)
-                self.save()
-
+        assert self.ready_to_publish
+        self.published_date = timezone.now()
+        if not self.expiration_date:
+            self.expiration_date = self.published_date + timedelta(60)
+        self.save()
 
     def __unicode__(self):
         return "{0}, {1}".format(self.title, self.company)
 
+
+class Meetup(models.Model):
+
+    MEETUP = 'MEET'
+    CONFERENCE = 'CONF'
+    WORKSHOP = 'WORK'
+    MEETUP_TYPES = (
+        (MEETUP, 'meetup'),
+        (CONFERENCE, 'conference'),
+        (WORKSHOP, 'workshop'),
+    )
+
+    title = models.CharField(max_length=500)
+    organisation = models.ForeignKey(
+        Company,
+        related_name="meetups",
+        blank=True,
+        null=True,
+    )
+    type = models.CharField(max_length=4, choices=MEETUP_TYPES, default=MEETUP)
+    contact_email = models.EmailField(max_length=254)
+    city = models.CharField(max_length=100)
+    country = CountryField()
+    description = models.TextField()
+    is_recurring = models.BooleanField(
+        default=False,
+        help_text="Is your meetup recurring?"
+    )
+    #TODO this field should be required if the is_recurring is True
+    recurrence = models.CharField(max_length=100, blank=True, null=True)
```

Comment:
unless we really need/want to restrict the length of the CharFields, let's just set max_length on all of them to 255 (the actual max length) to avoid unnecessary exceptions (also applies to city and contact_email, and probably some others in the job model)